cookbooks/bach_sqoop/README.md

```diff
@@ -0,0 +1,30 @@
+Overview
+========
+
+This cookbook installs Apache Sqoop and needed JDBC jars. The Sqoop packages come from Hortonworks via 'apt'.  JDBC jars are downloaded directly.
+
+Requirements
+============
+
+This cookbook has been tested with Chef 11 and 12.
+
+Recipes
+=======
+
+Only a default recipe is included.  The default recipe installs Sqoop, creates its library directory, and installs any JDBC jars to the library directory.
+
+Attributes
+==========
+
+<ol>
```

Comment:
Mark-up supports [bulleted lists](https://guides.github.com/features/mastering-markdown/) on its own, no need to HTML.